src/connectors/cisco_umbrella.py

```diff
@@ -85,6 +85,7 @@ def get_data(organization_id: str, key: str, secret: str, params: dict = {}) ->
 def connect(connection_name, options):
     table_name = f'cisco_umbrella_{connection_name}_connection'
     landing_table = f'data.{table_name}'
+    options['organization_id'] = int(options['organization_id'])
```

Comment:
:+1:

Reply:
Does this break backwards compatibility with a connector set up before this patch? Or do the options get treated the same once they get yaml-ized and stored in the table comment?

Just double checking.

Reply:
yaml treats strings and int differently, just like JSON does, so the comments in currently deployed connectors should be updated. don't think anyone else is using this one just yet, but in future we should include this in migration notes.